src/main/scala/com/fortysevendeg/scala/android/ui/pathmorphing/Layout.scala

```diff
@@ -142,30 +142,32 @@ trait Layout
                 ) <~ tableLayoutRowStyle
               ) <~ tableLayoutStyle <~ wire(colorSelectorGroup),
               w[TextView] <~ wire(strokeTitle) <~ tvText(R.string.title_select_stroke) <~ titleStyle,
-              w[SeekBar] <~ wire(strokeSelector) <~ strokeStyle <~ sbOnSeekBarChangeListener(
-                OnSeekBarChangeListenerHandler(
-                  onProgressChangedHandler = (view: SeekBar, progress: Int, fromUser: Boolean) => {
+              w[SeekBar] <~ wire(strokeSelector) <~ strokeStyle <~ MultFuncOn.SeekBarChange[SeekBar](
+                (onProgressChanged: OverrideMethodType) => {
+                  (view: SeekBar, progress: Int, fromUser: Boolean) => {
                     val stroke = progress + 1
 
-                    (strokeTitle <~ tvText(context.get.getResources().getString(R.string.title_select_stroke, stroke.toString))) ~
+                    (strokeTitle <~ tvText(context.get.getResources.getString(R.string.title_select_stroke, stroke.toString))) ~
                         (icon <~ pmdStroke(stroke dp))
                   }
-                )
+                },
+                (onStartTrackingTouch: OverrideMethodType) => (seekBar: SeekBar) => Ui(Some(seekBar)),
```

Comment:
At this moment you must specify all the methods to override. That adds an additional difficulties :(